Makefile

```diff
@@ -18,6 +18,19 @@ INST_PKG_TMP = pkg_temp
 API_SYNC_SOURCE = pkg/inst/context.go
 API_SYNC_TARGET = tool/internal/instrument/api.tmpl
 
+# License header check exclusions
+LICENSE_EXCLUDE_PATHS := -path '**/vendor/*' \
```

Comment:
Let's exclude .github/workflows and the configuration files. We don't actually need license headers for configuration files.

Reply:
Sorry, please give me a moment while I submit the code. 😬